website/docs/docs/build/fill_nulls_guide.md

```diff
@@ -0,0 +1,89 @@
+# Filling null values for simple and derived/ratio metrics
+
+You may want to coalesce null metric values with another value, like zero. This is useful if you want to express a numeric value for a metric for every row of data returned.
+
+The `fill_nulls_with` allows you to coalesce null values with a chosen integer, however, due to the implementation of derived metrics, you can still encounter null values in a query even if the `fill_nulls_with` parameter is set on the input metrics. This guide walks through how to use fill null values for simple metrics, and how to use `join_to_timespine` and `fill_nulls_with` together to ensure there are no null values in derived and ratio metrics.  
+
+### Filling null values for simple metrics
+
+Say you have three metrics: website `visits`, `leads`, and a derived metric called `leads_to_website_visit` that calculates the ratio of leads to site visits. There could be days when there are site visits, but none of these site visits converted into leads. You may want to display 0 as the value for leads on days when there are no conversions. 
+
+You can do this by adding the `fill_nulls_with` parameter to the measure input on the leads metric. 
+
+```sql
```

Comment:
Does it matter that this says `sql` instead of `yaml`?

Reply:
agree - have changed it to yaml thank you!